membership-attribute-service/app/components/TouchpointComponents.scala

```diff
@@ -102,10 +96,9 @@ class TouchpointComponents(
 
   lazy val zuoraSoapService = {
     lazy val zuoraSoapClient =
-      new ClientWithFeatureSupplier(
+      new Client(
         apiConfig = backendConfig.zuoraSoap,
         httpClient = RequestRunners.configurableFutureRunner(timeout = Duration(30, SECONDS)),
-        extendedHttpClient = RequestRunners.futureRunner,
```

Comment:
we had an http client and an extendedHttpClient but they were both the same in members-data-api.
The extended one has the default (shorter, iirc 10 seconds) timeout, which seems counterintuitive and I don't think it was intentional.
The separation was introcued in this pr to increase the timeout for createSubscription (which was used by subscribe and membership)
https://github.com/guardian/membership-common/pull/322
and pulled into this repo in bulk and with identcal parameter for each
https://github.com/guardian/members-data-api/pull/132/files#diff-246f8135447df9219beb681d466ff8f39bd2afae1699a967ec880ef97a2d39e9R59-R60